src/main/kotlin/no/nav/familie/tilbake/behandling/batch/AutomatiskSaksbehandlingTask.kt

```diff
@@ -14,18 +25,52 @@ import java.util.UUID
     maxAntallFeil = 3,
     triggerTidVedFeilISekunder = 60 * 5L,
 )
-class AutomatiskSaksbehandlingTask(private val automatiskSaksbehandlingService: AutomatiskSaksbehandlingService) : AsyncTaskStep {
+class AutomatiskSaksbehandlingTask(
+    private val automatiskSaksbehandlingService: AutomatiskSaksbehandlingService,
+    private val kravgrunnlagService: KravgrunnlagService,
+    private val behandlingRepository: BehandlingRepository,
+    private val featureToggleService: FeatureToggleService,
+) : AsyncTaskStep {
     private val logger = LoggerFactory.getLogger(this::class.java)
 
     override fun doTask(task: Task) {
         logger.info("AutomatiskSaksbehandlingTask prosesserer med id=${task.id} og metadata ${task.metadata}")
         val behandlingId = UUID.fromString(task.payload)
+
+        if (kravgrunnlagService.sumFeilutbetalingsbeløpForBehandlingId(behandlingId) > Constants.FIRE_X_RETTSGEBYR) {
```

Comment:
Ser at dette ikke er noe problem i praksis, men skulle vi hatt med AUTOMATISK_BEHANDLE_TILBAKEKREVING_UNDER_4X_RETTSGEBYR i denne if-sjekken? 

Kan vel ikke se for meg at vi øker grensen for de kravene som nå er hhv. 500 og 1/2 rettsgebyr, men... 🤔

Reply:
Kanskje unne oss en TODO: sjekk med faktisk beløp saksbehandler har sett i simulering?

Reply:
Er det ikke fint at denne valideringen gjelder for begge typer automatisk behandling?

Ser det er notert i favro-kortet at beløpet i kravgrunnlaget skal sjekkes mot det saksbehandler så i simuleringen